Dockerfile

```diff
@@ -1,12 +1,27 @@
-FROM python:2.7
-RUN apt-get update && apt-get install -y libmysqlclient-dev gettext libjpeg62-turbo-dev
+FROM debian:jessie
+
+EXPOSE 8000
+CMD ["./bin/run-prod.sh"]
+
+RUN adduser --uid 1000 --disabled-password --gecos '' --no-create-home webdev
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends build-essential python python-dev python-pip \
+                                               libmysqlclient-dev libxslt1.1 libxml2 libxml2-dev libxslt1-dev && \
+    rm -rf /var/lib/apt/lists/*
 
 WORKDIR /app
 
+# Pin a known to work with peep pip version.
+RUN pip install pip==6.0.0
+
 # First copy requirements.txt and peep so we can take advantage of
 # docker caching.
-COPY requirements /tmp/requirements
```

Comment:
need to keep the copy of the `requirements` dir since `requirements.txt` below requires those files as well. The docker build isn't working for me due to that.